kitsune/community/templates/community/email/first_answer.html

```diff
@@ -0,0 +1,76 @@
+            forums! You just made somebody's day!
+        {% endtrans %}
+    </p>
+
+    <p>
+        {% trans %}
+            Mozilla support is fully powered by our volunteer community that is
+            helping hundreds of users everyday. This community is formed by
+            passionate Firefox and Thunderbird users who take some of their
+            spare time to help others. We couldn't do it without your passion
+            and enthusiasm. Your contribution, as minor as it might seem, has
+            tremendous value. Solving one issue in the support forums can help
+            up to 1000 users a day!
+        {% endtrans %}
+    </p>
+
+    <p><strong>{{ _('Tell us about yourself!') }}</strong></p>
+
+    <p>
+        {% trans intro_url=host + '/en-US/forums/contributors', irc_url='https://client01.chat.mibbit.com/?server=irc.mozilla.org&channel=#sumo' %}
+            We are always very happy to hear from new people who like helping
+            on the forums so If you haven’t met our community yet
+            <a href="{{ intro_url }}">
+                please take a minute to introduce yourself
+            </a>.
+            If you want to stay low-profile, that’s also ok :). You can also
+            find us all hanging out on IRC, on the
+            <a href="{{ irc_url }}">
+                #sumo channel
+            </a>.
+        {% endtrans %}
+    </p>
+
+    <p><strong>{{ _('Learn new things') }}</strong></p>
+
+    <p>
+        {% trans training_url=host + '/en-US/kb/introduction-contributor-quality-training?utm_source=training&utm_medium=1intro&utm_term=cqt&utm_campaign=Contributorqualitytraining' %}
+            If you are interested in learning more about how this whole support
+            thing works, check out our
+            <a href="{{ training_url }}">
+                contributor training
+            </a>.
+            This includes some cool troubleshooting info, guidelines, how to
+            questions, and info about where to find more help if stuck.
+        {% endtrans %}
+    </p>
+
+    <p><strong>{{ _('Stay in touch') }}</strong></p>
+
+    <p>
+        {% trans forums_url=host + '/forums/contributors', blog_url='http://blog.mozilla.org/sumo/', twitter_url='http://twitter.com/sumo_mozilla', meetings_url='https://wiki.mozilla.org/Support/Weekly_Meetings' %}
+            Finally, if you want to follow all news related to SUMO, don’t
+            forget about our
+            <a href="{{ forums_url }}">community forums</a>,
+            <a href="{{ blog_url }}">official blog</a> and
+            <a href="{{ twitter_url }}">Twitter account</a>.
+            We also
+            <a href="{{ meetings_url }}">
+                meet almost every Monday
+            </a> - and you can join us.
+        {% endtrans %}
+    </p>
+
+    <p>
+        {% trans %}
+            Thank you once more. We are looking forward to seeing you more often
+            around SUMO! Keep rocking!
```

Comment:
I think this is awkward phrasing and should be "looking forward to seeing you around SUMO more often!"